gdx/src/com/badlogic/gdx/graphics/g2d/Sprite.java

```diff
@@ -529,7 +529,17 @@ public float getX () {
 	public float getY () {
 		return y;
 	}
-
+	
+	/** @return the center x of the sprite, not accounting for scale. */

```

Comment:
I don't think origin affects the x and y positions at all. The x and y properties always refer to the bottom left corner not taking into account scaling. Frequently I make the mistake of not setting the origin of my sprites to the center and I never notice it until I try to rotate them - then I see they are rotating around the bottom left corner. When I change the origin it never affects the positioning.

Why someone put in the setCenter methods without the getCenter is beyond me.